scopus-import/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -1624,23 +1618,28 @@ private OrganizationTp createOrganization(List<Serializable> organizationName) {
         return organization;
     }
 
-    private void checkAuthorOrcidAndSequenceNumber(AuthorTp authorTp, List<Contributor> contributors) {
+    private void assertIsSameAuthor(AuthorTp authorTp, List<Contributor> contributors) {
         if (nonNull(authorTp.getOrcid())) {
             var orcidAsUriString = getOrcidAsUriString(authorTp);
             var optionalContributor = findContributorByOrcid(orcidAsUriString, contributors);
             assertTrue(optionalContributor.isPresent());
             var contributor = optionalContributor.get();
-            assertEquals(authorTp.getSeq(), contributor.getSequence().toString());
+            assertThat(contributor.getIdentity().getName(), containsString(authorTp.getSurname()));
```

Comment:
Comparing sequence is. bit weird, but name should be the same

Reply:
I remember struggling to define what defines an authorTp equalTo nva-contributor. I vaguely remember you could have theoretically two people named the same with a different sequence number on a publication. Rare but it happens.